Replicator/Replicator.cc

```diff
@@ -492,6 +487,7 @@ namespace litecore { namespace repl {
             notifyEndedDocuments();
             _delegate->replicatorConnectionClosed(this, status);
         }
+        _weakConnectionDelegateThis = nullptr;
```

Comment:
This is an edge case ...  What if a replicator is created but is released without start, will it cause a memory leak?

Reply:
It's an excellent point. Must be paired. Will fix it. I need to find a solid pair.